docker-compose/config/worker-ubuntu22-04.jsonnet

```diff
@@ -1,22 +1,80 @@
 local common = import 'common.libsonnet';
 
 {
-  blobstore: common.blobstore,
+  blobstore: {
+    actionCache: common.blobstore.actionCache,
+    contentAddressableStorage: {
+      readCaching: {
+        slow: common.blobstore.contentAddressableStorage,
+        fast: {
+            'local': {
+                keyLocationMapOnBlockDevice: {
+                file: {
+                    path: '/storage-worker-cas/key_location_map',
+                    sizeBytes: 100 * 1024 * 1024,
+                },
+                },
+                keyLocationMapMaximumGetAttempts: 8,
+                keyLocationMapMaximumPutAttempts: 32,
+                oldBlocks: 8,
+                currentBlocks: 24,
+                newBlocks: 1,
+                blocksOnBlockDevice: {
+                  source: {
+                      file: {
+                      path: '/storage-worker-cas/blocks',
+                      sizeBytes: 8 * 1024 * 1024 * 1024,
+                      },
+                  },
+                  spareBlocks: 3,
+                  dataIntegrityValidationCache: {
+                      cacheSize: 50000,
+                      cacheDuration: '14400s',
+                      cacheReplacementPolicy: 'LEAST_RECENTLY_USED',
+                  },
+                },
+                persistent: {
+                stateDirectoryPath: '/storage-worker-cas/persistent_state',
+                minimumEpochInterval: '300s',
+                },
+            },
+        },
+        replicator: { 'local' : {} },
+      },
+    },
+  },
   browserUrl: common.browserUrl,
   maximumMessageSizeBytes: common.maximumMessageSizeBytes,
   scheduler: { address: 'scheduler:8983' },
   global: common.global,
   buildDirectories: [{
-    native: {
-      buildDirectoryPath: '/worker/build',
-      cacheDirectoryPath: '/worker/cache',
-      maximumCacheFileCount: 10000,
-      maximumCacheSizeBytes: 1024 * 1024 * 1024,
-      cacheReplacementPolicy: 'LEAST_RECENTLY_USED',
+    // native: {
+    //   buildDirectoryPath: '/worker/build',
+    //   cacheDirectoryPath: '/worker/cache',
+    //   maximumCacheFileCount: 600000,
+    //   maximumCacheSizeBytes: 1024 * 1024 * 1024 * 8,
+    //   cacheReplacementPolicy: 'LEAST_RECENTLY_USED',
+    // },
+    virtual: {
+    // Perform builds in a virtual mount (e.g., FUSE) 
+    // Use if a lot of time spent on builds is on fetching inputs
+      maximum_execution_timeout_compensation: "3600s",
```

Comment:
Could you use `maximumExecutionTimeoutCompensation` here? Also be sure to use `camelCase` consistently.